DigitalLearningSolutions.Web.Tests/Helpers/CustomClaimHelperTests.cs

```diff
@@ -152,5 +152,21 @@ public void GetCustomClaimAsRequiredInt_handles_claim_invalid()
             // Then
             Assert.Throws<FormatException>(() => user.GetCustomClaimAsRequiredInt(CustomClaimTypes.LearnCandidateId));
         }
+
+        [Test]
+        public void GetCandidateIdHandlesZeroAsNull()

```

Comment:
Test looks good but can you change the name to match the format of the rest of them

Reply:
Fixed